src/components/column-sorting/column-sorting.component.ts

```diff
@@ -23,6 +23,10 @@ export class ColumnSortingComponent implements OnInit, OnChanges, OnDestroy {
     /** Determine if a column can have a `NoSort` state */
     @Input() allowNoSort: boolean = true;
 
+    /** Specifies name of the ascending and descending icon */
```

Comment:
You need to comment each property in order to document them correctly. This is a formal comment syntax which has a specific structure in order to associate documentation with API features. Read up on it here:
https://github.com/microsoft/tsdoc